onnxruntime/core/providers/vsinpu/builders/impl/activation_op_builder.h

```diff
@@ -21,6 +21,8 @@
  *    DEALINGS IN THE SOFTWARE.
  *
  *****************************************************************************/
+#ifndef ONNXRUNTIME_CORE_PROVIDERS_VSINPU_BUILDERS_IMPL_ACTIVATION_OP_BUILDER_H_
```

Comment:
How about `#pragma once`? That is consistent with other provider and has same effect.

Reply:
OK，I‘ll modify all headers with #pragma once

Reply:
Done.